client/dashboard/profile-wizard/steps/theme/index.js

```diff
@@ -92,7 +92,7 @@ class Theme extends Component {
 			location,
 		} );
 
-		if ( theme !== activeTheme && getPriceValue( price ) <= 0 ) {
+		if ( slug !== activeTheme && getPriceValue( price ) <= 0 ) {
```

Comment:
Nice fix here!  We may want to check the `is_installed` property of the theme as there may be already installed themes in the list besides just the active theme.

Reply:
Thank you @joshuatf, good idea! I added this check and the method call in the commit [ce90633](https://github.com/woocommerce/woocommerce-admin/pull/4148/commits/ce906338f47d1330f767756a50d7fa28a5972d37)
Could you take a look again at this PR?